Bug Report: tx throttler panics with target_replication_lag_sec of 1 (NewTicker(0))

### Overview of the Issue

When the transaction throttler is enabled with a `target_replication_lag_sec` of `1`, `vttablet` panics shortly after startup:

```
panic: non-positive interval for NewTicker

goroutine [running]:
time.NewTicker(...)
vitess.io/vitess/go/vt/vttablet/tabletserver/txthrottler.(*txThrottlerStateImpl).updateMaxLag(...)
	go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go:377
```

The `updateMaxLag` goroutine builds its ticker from `TargetReplicationLagSec / 2`:

https://github.com/vitessio/vitess/blob/af1642e3f7085781597932cb5ba227ad1f3e358a/go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go#L376-L377

Because `TargetReplicationLagSec` is an integer, a value of `1` makes `1 / 2 == 0`, so the code calls `time.NewTicker(0)`, and `time.NewTicker` panics on any non-positive duration.

The value `1` is a *valid* configuration: `verifyTxThrottlerConfig` only rejects it through `MaxReplicationLagModuleConfig.Verify()`, which requires `target_replication_lag_sec >= 1`:

https://github.com/vitessio/vitess/blob/af1642e3f7085781597932cb5ba227ad1f3e358a/go/vt/throttler/max_replication_lag_module_config.go#L85-L87

So a config that passes validation still crashes the tablet.

### Reproduction Steps

1. Start `vttablet` with the transaction throttler enabled and a target lag of 1 second, e.g.:
   ```
   --enable-tx-throttler --tx-throttler-config "target_replication_lag_sec: 1 max_replication_lag_sec: 2 initial_rate: 100 max_increase: 1 emergency_decrease: 0.5 min_duration_between_increases_sec: 40 max_duration_between_increases_sec: 62 min_duration_between_decreases_sec: 20 spread_backlog_across_sec: 20 age_bad_rate_after_sec: 180 bad_rate_increase: 0.1 max_rate_approach_threshold: 0.9" --tx-throttler-tablet-types replica --tx-throttler-healthcheck-cells zone1
   ```
2. The config passes verification and the tablet starts opening the throttler.
3. The `updateMaxLag` goroutine panics with `non-positive interval for NewTicker`.

### Binary Version

```
main @ af1642e3f7085781597932cb5ba227ad1f3e358a
```

### Operating System and Environment details

Reproduces on any platform; the panic is in platform-independent Go code.

### Log Fragments

```
panic: non-positive interval for NewTicker
```

--- a/go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go
+++ b/go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go
@@ -374,7 +374,7 @@ func (ts *txThrottlerStateImpl) throttle() bool {
 func (ts *txThrottlerStateImpl) updateMaxLag() {
 	defer ts.waitForTermination.Done()
 	// We use half of the target lag to ensure we have enough resolution to see changes in lag below that value
-	ticker := time.NewTicker(time.Duration(ts.config.TxThrottlerConfig.TargetReplicationLagSec/2) * time.Second)
+	ticker := time.NewTicker(time.Duration(ts.config.TxThrottlerConfig.TargetReplicationLagSec) * time.Second / 2)
 	defer ticker.Stop()
 outerloop:
 	for {

--- a/go/vt/vttablet/tabletserver/txthrottler/tx_throttler_test.go
+++ b/go/vt/vttablet/tabletserver/txthrottler/tx_throttler_test.go
@@ -187,6 +187,59 @@ func TestEnabledThrottler(t *testing.T) {
 	assert.Zero(t, throttlerImpl.throttlerRunning.Get())
 }
 
+// TestUpdateMaxLagTickerLowTargetLag verifies that a target replication lag of 1
+// second, which passes config verification, does not make the updateMaxLag
+// goroutine build a time.NewTicker with a non-positive interval and panic the
+// tablet. The old code computed TargetReplicationLagSec/2, which rounds down to 0
+// for a target of 1.
+func TestUpdateMaxLagTickerLowTargetLag(t *testing.T) {
+	ctx := t.Context()
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	defer resetTxThrottlerFactories()
+	ts := memorytopo.NewServer(ctx, "cell1")
+
+	mockHealthCheck := NewMockHealthCheck(mockCtrl)
+	mockHealthCheck.EXPECT().Subscribe("TxThrottler").AnyTimes()
+	mockHealthCheck.EXPECT().RegisterStats().AnyTimes()
+	mockHealthCheck.EXPECT().Close().AnyTimes()
+	healthCheckFactory = func(ctx context.Context, topoServer *topo.Server, cell, keyspace, shard string, cellsToWatch []string) (discovery.HealthCheck, error) {
+		return mockHealthCheck, nil
+	}
+
+	mockThrottler := NewMockThrottler(mockCtrl)
+	mockThrottler.EXPECT().UpdateConfiguration(gomock.Any(), true /* copyZeroValues */).AnyTimes()
+	mockThrottler.EXPECT().MaxLag(gomock.Any()).Return(uint32(0)).AnyTimes()
+	mockThrottler.EXPECT().Close().AnyTimes()
+	throttlerFactory = func(name, unit string, threadCount int, maxRate int64, maxReplicationLagConfig throttler.MaxReplicationLagModuleConfig) (throttler.Throttler, error) {
+		return mockThrottler, nil
+	}
+
+	cfg := tabletenv.NewDefaultConfig()
+	cfg.EnableTxThrottler = true
+	cfg.TxThrottlerTabletTypes = &topoproto.TabletTypeListFlag{topodatapb.TabletType_REPLICA}
+	// A target replication lag of 1s is accepted by Verify (it only rejects < 1).
+	cfg.TxThrottlerConfig.Get().TargetReplicationLagSec = 1
+
+	env := tabletenv.NewEnv(vtenv.NewTestEnv(), cfg, t.Name())
+	throttler := NewTxThrottler(env, ts)
+	throttlerImpl, ok := throttler.(*txThrottler)
+	require.True(t, ok)
+	throttler.InitDBConfig(&querypb.Target{
+		Cell:     "cell1",
+		Keyspace: "keyspace",
+		Shard:    "shard",
+	})
+
+	// Open starts the updateMaxLag goroutine, which builds the ticker immediately.
+	// Close blocks until that goroutine has terminated, so a non-positive ticker
+	// interval panics within this test rather than being lost in a background crash.
+	require.NoError(t, throttlerImpl.Open())
+	throttler.Close()
+	assert.Zero(t, throttlerImpl.throttlerRunning.Get())
+}
+
 func TestFetchKnownCells(t *testing.T) {
 	ctx := t.Context()
 	{

